resources/files/rbconfig-update.rb

```diff
@@ -0,0 +1,44 @@
+# **********************
+# rbconfig-update.rb
+#
+# This script is intended to write a new rbconfig file that is
+# a copy of an existing rbconfig with updated entries.
+#
+# entries to update should passed in as a string that can be
+# evaluated as a ruby array in the first argument to the script
+#
+# **********************
+
+# replace_line
+#
+# The following replaces any configuration line in an rbconfig
+# that matches any of the key value pairs listed in the CHANGES
+# hash
+def replace_line(changes, line, file)
+  changes.each do |change_key, change_value|
+    if line.strip.start_with?("CONFIG[\"#{change_key}\"]")
+      puts "Updating #{change_key} to CONFIG[\"#{change_key}\"] = \"#{change_value}\""
+      file.puts line.sub(/CONFIG.*/, "CONFIG[\"#{change_key}\"] = \"#{change_value}\"")
```

Comment:
I think I'd rather have this method return the string (either as-is or changed), so that all the prints are centralized below